Library/Formula/mongrel2.rb

```diff
@@ -13,6 +13,8 @@ class Mongrel2 < Formula
     sha256 "7cdef65628b457f03f3eb76fcf74276ffbc4b7d9fa0ff6d017fa43a8db98eeca" => :mavericks
   end
 
+  patch :DATA
```

Comment:
You can use a patch with https://github.com/mongrel2/mongrel2/commit/7cb8532e2ecc341d77885764b372a363fbc72eff.patch instead of embedding it.